doc/ysh-faq.md

```diff
@@ -72,22 +72,24 @@ The rules are subtle, so we avoid inventing new ones.
 
 ## How do I write the equivalent of `echo -e` or `echo -n`?
 
-YSH removed these flags to `echo` because it has good alternatives
+YSH removed these flags to `echo` to solve their interference problem with
+variable substitutions (see next) and double-quoted backslash.
 
-To escape variables, use the statically-parsed string language, rather than
+To echo escape sequences, use a statically-parsed string literal, rather than
 `echo -e`:
 
-    echo $'tab \t newline \n'   # YES
-    echo j"tab \t newline \n"   # TODO: J8 notation
+    echo $'tab: \t newline: \n' # YES
+    echo u'tab: \t A: \u{41}'   # J8 unicode notation
```

Comment:
I think this should be exactly the same string with `newline: \n`, and we could possibly link to `doc/ref` to show the full syntax

Reply:
Maybe newline \n as unicode? I think it's good to include some general idea of differences.

Reply:
`echo u'tab: \t newline: \u{A}'`

Reply:
include idea of usages rather